CryptoPrices/Sources/ContentView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ContentView.swift
+//  CryptoPrices
+//
+//  Created by Doan Thieu on 28/11/2022.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+
+    var body: some View {
+        VStack {
+            Text("Crypto Prices")
```

Comment:
Do you think we should add `TODO: -` to remind us to remove it later?

Reply:
Normally the `TODO` means we can put it off, for now, and we can come back to address it later. But I think this is not necessary since this is kind of __must do__.
Also, we can look at it as the _current UI design_ from the designer, not the placeholder one 😅